cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -176,13 +176,19 @@ public static void editConfig()throws Exception{
             else
             if(id==3){ide="idea";System.out.println("\n--------Selected IDE:IntelliJ_IDEA--------");break;}
             else
+            if(id==4){System.out.println("\n--------Enter Launch command for the editor/ide--------");ide=reader.readLine();break;}
+            else
             System.out.println("\n--------Invalid Input!! Try Again--------");
             }
 
             //-----------Now we'll get preferred clone path on local file system from user--------------
             while(true){
             System.out.println("\n--------Enter preferred Clonning path--------");
             clone_path=reader.readLine();
+            // If user enters path in form of ~/directory/sub_directory
+            if(clone_path.startsWith("~"))
```

Comment:
this will fix it when entering, but not if user puts it directly in the file.

Reply:
Thanks for pointing that out.
I'll resolve that soon :)

Reply:
@maxandersen 
> this will fix it when entering, but not if user puts it directly in the file.

I've resolved that.
Please check out :)

Reply:
i don't see that fix ? seem to still be converting ~ to user.home at add rather than do it when reading ?